authorization/invitation/service/invitation_service.go

```diff
@@ -253,7 +253,7 @@ func (s *invitationServiceImpl) processTeamInviteNotifications(ctx context.Conte
 	var messages []notification.Message
 
 	for _, n := range notifications {
-		acceptURL := fmt.Sprintf("%s/api/invitations/accept?code=%s", s.config.GetAuthServiceURL(), n.invitation.AcceptCode.String())
+		acceptURL := fmt.Sprintf("%s/api/invitations/accept/%s", s.config.GetAuthServiceURL(), n.invitation.AcceptCode.String())
```

Comment:
can we have a constant for `/api/invitations/accept/`, so it does not need to be duplicated twice here and once more in https://github.com/fabric8-services/fabric8-auth/pull/607/files#diff-bdfa221aeccbdd7f5ee03b7c859c6c2dR47 ?

Reply:
well done with the URL fix!